printer_mods/khay/micro_backplate/README.md

```diff
@@ -0,0 +1,10 @@
+Scrolling through discord, there a V0 owners like me that experienced really hot electronics or motors because of the enclosed back of the printer.
+As a serious aduld without kids or furry kids, you could leave the backplate off the printer to allow for airflow for the electronics/motors. 
+Unfortunately this will result in loosing hot air from the printchamber. 
+So in addition to Weaslus "Electronics-Bay-Separator" or the V0.1 revision motor panel i have sketched this "Micro Backplate" out of the original
+corners for the backplate. It is split in two parts to fit on a spec V0. 
+
```

Comment:
Include your image in the README, like this

    ![Backplate](DSC_2959.jpg)

Reply:
done